control/views.py

```diff
@@ -210,8 +210,7 @@ def temperature_converter(request):
                 elif unit_to == 'fahrenheit':
                     calculations = (unit_value * 9 / 5) - 459.67
 
-            else:
-                result = '{} {}'.format(calculations, unit_to)
+            result = '{} {}'.format(calculations, unit_to)
```

Comment:
En realidad, aunque sea lo mismo y parezca que escribes menos, es mas legible y explicito con el else, por lo que no se aconseja quitar.

Reply:
El problema es que nunca entra en ese else y cuando conviertes no da resultado.

Reply:
Ah, ya veo. Si, en este caso makes sense. 

Por cierto, linea 190 y 196 se pisan.

Reply:
good point, no se en que estaba pensando ahi :D

Reply:
Yo si... 😞

Reply:
not anymore my friend